andela/kissa-rc
# #137005187 Multivendor/Marketplace

#### What does this PR do?
It implements a multivendor shop feature on the app
#### Description of Task completed?
1. Added sign-up as vendor feature
2. filtered orders based on vendor's products
3. created a customized shop for each vendor
4. updated static pages to show per vendor shop
#### How should this be manually tested?
1. visit the link on heroku
2. signup as a vendor
3. get activated by the admin
4. Create Products and manage them.
5. Use guest login to buy products and visit shop
5. view your orders to check filter
#### Any background context you want to provide?
Nil
#### What are the relevant pivotal tracker stories?
[Multivendor/Marketplace](https://www.pivotaltracker.com/story/show/137005187)
#### Screenshots (if appropriate)
![screen shot 2017-02-13 at 8 14 13 pm](https://cloud.githubusercontent.com/assets/23451840/22898901/425d9ee0-f229-11e6-9974-1afd96729b93.png)

#### Questions:
Nil
